dpti/workflows/simulations/equi_sim.py

```
@@ -0,0 +1,457 @@
+                file_path=self.NODEDATA_FILENAME,
+                file_content=self.node_settings.model_dump_json(indent=4),
+            )
+
+            return {"current_produced_paths": io_handler.current_produced_paths}
+
+    def _run(self) -> str:
+        submission_r = self.workflow_services.job_executor.submit(job_dir=self.job_dir)
+        return submission_r
+
+    # @task
+    def _extract(self) -> Dict[str, Any]:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)